compiler/src/main/scala/org/scalajs/core/compiler/GenJSCode.scala

```diff
@@ -706,7 +706,8 @@ abstract class GenJSCode extends plugins.PluginComponent
 
       def lambda(params: List[js.ParamDef], body: js.Tree)(
```

Comment:
I think this name is worth changing. It's not a simple `lambda` anymore. It's actually a `thisFunctionLambda` something. Maybe `memberLambda`?

Reply:
`memberLambda` sounds fine :)